src/routes/tools_routes.js

```diff
@@ -16,92 +15,23 @@ import carddb, {
   getRelatedCards,
 } from '../util/carddb';
 const cardutil = require('../client/utils/cardutil');
-const { SortFunctionsOnDetails, ORDERED_SORTS } = require('../client/utils/Sort');
-const { makeFilter, filterCardsDetails } = require('../client/filtering/FilterCards');
+const { makeFilter } = require('../client/filtering/FilterCards');
 const generateMeta = require('../util/meta');
 const util = require('../util/util');
 const { csrfProtection } = require('./middleware');
 const { handleRouteError, render, redirect } = require('../util/render');
 
-const { PrintingPreference } = require('../datatypes/Card');
 const CardHistory = require('../dynamo/models/cardhistory');
 const Cube = require('../dynamo/models/cube');
 
+const { searchCards } = require('../util/tools');
```

Comment:
Not sure if I like putting the common stuff in util. Since its used in both top cards and card search, which logically should be two different route files, it needs a common place.